conda-forge.yml

```diff
@@ -1,2 +1,5 @@
 max_py_ver: '37'
-max_r_ver: '35'
\ No newline at end of file
+max_r_ver: '35'
+channels:
+  targets:
+    - ["conda-forge", "testing"]
```

Comment:
You shouldn't change this file. You need to add a conda_build_config file to the recipe. See conda-smithy documentation

Reply:
Oops, sorry about that. Thanks!